test/components/scroll-spy.spec.coffee

```diff
@@ -0,0 +1,61 @@
+{React, Testing, expect, _, sinon} = require './helpers/component-testing'
+
+ScrollSpy = require '../../src/components/scroll-spy'
+
+
+NestedComponent = React.createClass
+  render: -> React.createElement('div')
+
+RootComponent = React.createClass
+  render: ->
+
+    React.createElement('div', {},
+      React.createElement(ScrollSpy, @props,
+        React.createElement(NestedComponent, onScreenElements:[])
+      )
+    )
```

Comment:
Should we make this file a `.cjsx` instead and use the syntax we are using everywhere for React elements?